src/electionguard_cli/steps/shared/tally_step.py

```diff
@@ -0,0 +1,55 @@
+from typing import Iterable, List, Tuple
+from electionguard.scheduler import Scheduler
+from electionguard.data_store import DataStore
+from electionguard.tally import CiphertextTally
+from electionguard.ballot_box import get_ballots
+from electionguard.ballot import BallotBoxState, SubmittedBallot
+
+from electionguard_cli.cli_models import BuildElectionResults
+from .cli_step_base import CliStepBase
+
+
+class TallyStep(CliStepBase):
+    """Responsible for creating a tally and retrieving spoiled ballots."""
+
+    def get_from_ballots(
+        self,
+        build_election_results: BuildElectionResults,
+        ballots: List[SubmittedBallot],
+    ) -> Tuple[CiphertextTally, List[SubmittedBallot]]:
+        self.print_header("Creating Tally")
+        tuble_ballots = [(None, b) for b in ballots]
```

Comment:
This code that converts a list of ballots into a list of tuples of ballots is awkward, but I was aiming to change the underlying python core as little as possible, and a list of tuples of ballots is what it currently takes, so I'm adapting this code to match.  I can change the source method if desired, I just worry about what other unintended consequences that may have.